daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -603,7 +619,19 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite)
       }
       case (None, Some(defSchema), None) => {
         Assert.invariant(model != "") // validation of the TDML should prevent this
-        EmbeddedSchemaSource(defSchema.xsdSchema, defSchema.name)
+
+        /* The file, line, and column attributes we add for tracking purposes
+         * cause issues with the IBM DFDL processor as it doesn't allow
+         * attributes that it doesn't understand, so we must remove them */
+        val processedSchema = {
+          if (isCrossTest(this.tdmlDFDLProcessorFactory.implementationName)) {
+            stripLineColInfo(defSchema.xsdSchema)
+          } else {
+            defSchema.xsdSchema
+          }
+        }
+
+        EmbeddedSchemaSource(processedSchema, defSchema.name)
```

Comment:
Can you add a test to make sure that when there is an SDE in a embeded and non-embedded schema that we get a line/col number when the implementation is "daffodil"?

Reply:
Also, can you double check with the [OpenDFDL/ibmDFDLCrossTester](https://github.com/OpenDFDL/ibmDFDLCrossTester)? One tests fails with ``sbt test`` with this PR with an error about unsupported attribute, so I think something still isn't being removed corrrectly.